frontend/src/metabase/static-viz/components/XYChart/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const MAX_ROTATED_TICK_WIDTH = 100;
 
-export const CHART_PADDING = 10;
+export const CHART_PADDING = 12;
```

Comment:
These 2 values (also `LABEL_OFFSET`) change is intended to fix the overflown y-axis label. I guess this is because there was a font size change but certain constants remain the same, so they somehow cut off the y-axis labels.

#### After
![image](https://user-images.githubusercontent.com/1937582/191037412-10bf72f1-0dcc-4fc0-ba9a-ff94589152bb.png)


#### Before (look at number 4)
![image](https://user-images.githubusercontent.com/1937582/191037370-5358088a-a249-4c1a-aac7-408396f44397.png)